### PR TITLE
Fix install issue for python 3.9

### LINKED ISSUE #3698: ERROR: Failed building wheel for scipy
``` 
gfortran:f77: scipy/sparse/linalg/eigen/arpack/ARPACK/UTIL/cvout.f
  gfortran:f77: scipy/sparse/linalg/eigen/arpack/ARPACK/UTIL/zvout.f
  gfortran:f77: scipy/sparse/linalg/eigen/arpack/ARPACK/UTIL/svout.f
  gfortran:f77: scipy/sparse/linalg/eigen/arpack/ARPACK/UTIL/dmout.f
  gfortran:f77: /private/var/folders/_y/2zwzc3xj3y3139p_k4sz867c0000gn/T/pip-install-dfvawfib/scipy_bbde78652187449da3ffd27b4bd17630/scipy/_build_utils/src/wrap_dummy_g77_abi.f
  error: Command "/usr/local/bin/gfortran -Wall -g -ffixed-form -fno-second-underscore -fPIC -O3 -funroll-loops -Iscipy/sparse/linalg/eigen/arpack/ARPACK/SRC -I/private/var/folders/_y/2zwzc3xj3y3139p_k4sz867c0000gn/T/pip-build-env-scgt02_1/overlay/lib/python3.9/site-packages/numpy/core/include -c -c scipy/sparse/linalg/eigen/arpack/ARPACK/SRC/cnaup2.f -o build/temp.macosx-10.15-x86_64-3.9/scipy/sparse/linalg/eigen/arpack/ARPACK/SRC/cnaup2.o" failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for scipy
Failed to build scipy
ERROR: Could not build wheels for scipy which use PEP 517 and cannot be installed directly
```

I am using python 3.9 and cannot install this.

After I did changes in two packages it worked.

```
diff --git a/setup.py b/setup.py
index 07ba58b1..721ee76d 100644
--- a/setup.py
+++ b/setup.py
@@ -46,9 +46,9 @@ setup(
         "jsonschema>=3.2.0",
         "google-cloud-storage==1.29.0",
         "elasticsearch==7.8.0",
-        "numpy==1.18.5",
+        "numpy==1.19.5",
         "python-ipmi==0.4.2",
-        "scipy==1.4.1",
+        "scipy==1.6.0",
         "PrettyTable==0.7.2",
         "azure-common==1.1.25",
         "azure-mgmt-compute==12.0.0",
```

### PR DESCRIPTION
**Pull request by petr-balogh**
Fixes #3698
Fixes: https://github.com/red-hat-storage/ocs-ci/issues/3768

Signed-off-by: Petr Balogh <pbalogh@redhat.com>

### CODE REVIEW

**Review by vasukulkarni — DISMISSED** [review]

**Review by vavuthu — DISMISSED** [review]

**Comment by petr-balogh** [comment]
Adding WIP to prevent merge as we need to wait till engineering will update their env to move to newer python.  @andrewschoen can you please comment here once you are ready?  Thanks

**Review by clacroix12 — APPROVED** [review]

**Comment by clacroix12** [comment]
@petr-balogh Not sure where this is defined (probably at the repo level) but it looks like we still need to disable the 3.6 PR check:

Screen Shot 2021-01-29 at 1 14 22 PM

**Comment by andrewschoen** [comment]
> Adding WIP to prevent merge as we need to wait till engineering will update their env to move to newer python. @andrewschoen can you please comment here once you are ready? Thanks

I just filed https://github.com/red-hat-storage/ocs-ci/pull/3783, I think we're good to go.

**Comment by petr-balogh** [comment]
I will coordinate merge of this on Monday and will look at pr check issue @clacroix12 . We need to first merge the MR in jenkins repo.

File: .github/workflows/pr_checks.yml [hunk 1]
```diff
@@ -7,7 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 3.6, 3.7 ]
+        python-version:
+          - 3.7
```
Comment (vasukulkarni) [inline .github/workflows/pr_checks.yml:11]: I think we should get rid of 3.7 to save test time

Comment (petr-balogh) [reply .github/workflows/pr_checks.yml:11]: Tests are running parallel so we will not save any time I think snd as we support 3.7, 3.8 and 3.9 I think it's still worth to test it. We can think of drop support for 3.7 later on but if we don't have any issue with it I would keep support it.

Comment (clacroix12) [reply .github/workflows/pr_checks.yml:11]: I think we should continue to execute unit tests on all versions of python we plan on supporting. We can't tell users we support a python version we don't test on.

**Review by jimdevops19 — APPROVED** [review]

**Pull request merged**